File: Sorting Algo/Insertion Sort/InsertionSort.c
#include<stdio.h>
void insertionSort(int * arr){
    int i,j;
    for(i=1;i<5;i++){
        int key=arr[i];
        j=i-1;
        while(j>=0 && arr[j]>key){
            arr[j+1]=arr[j];
            j--;
        }
        arr[j+1]=key;
    }
}
int main(){
    int arr[] = {2,1,3,53,5,6,46}
    insertionSort(arr);
    printf("Sorted array is : \n");
    for(int i=0;i<5;i++){
        printf("%d ",arr[i]);
    }
    return 0;
}

File: Sorting Algo/Insertion Sort/insertionSort.py
def insertionSort(arr):
    for i in range(1, len(arr)):
        key = arr[i]
        j = i-1
        while j >= 0 and key < arr[j]:
            arr[j+1] = arr[j]
            j -= 1
        arr[j+1] = key
    return arr

arr = {12,12,4,2,4,467}
insertionSort(arr)
print ("Sorted array is: \n")
for i in range(len(arr)):
    print ("%d" %arr[i])
